### PR TITLE
update type definition for debug hook

### PR DESCRIPTION
**Pull request by brane53**
- [x] Tell us about the problem your pull request is solving.

    **The type definition for the debug hook was missing the ***label*** and ***fieldNames*** parameters as specified in the docs.**

- [x] Are there any open issues that are related to this?

    **No**

- [x] Is this PR dependent on PRs in other repos?

    **No**

### CODE REVIEW

**Comment by eddyystop** [comment]
```
Installing to /home/travis/.npm/_npx/4816/lib/node_modules/dtslint/typescript-installs/3.2...
Installed!
Installing to /home/travis/.npm/_npx/4816/lib/node_modules/dtslint/typescript-installs/next...
Installed!
Error: /home/travis/build/feathers-plus/feathers-hooks-common/types/tests.ts:136:1
ERROR: 136:1  expect  TypeScript@next compile error: 
Expected 1-2 arguments, but got 0.
    at /home/travis/.npm/_npx/4816/lib/node_modules/dtslint/bin/index.js:163:19
    at Generator.next (<anonymous>)
    at fulfilled (/home/travis/.npm/_npx/4816/lib/node_modules/dtslint/bin/index.js:5:58)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! feathers-hooks-common@4.20.0 lint:types: `npx dtslint types`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the feathers-hooks-common@4.20.0 lint:types script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2019-01-10T07_04_52_206Z-debug.log
npm ERR! Test failed.  See above for more details.
The command "npm test" exited with 1.
Done. Your build exited with 1.
```

**Comment by eddyystop** [comment]
@j2L4e ^^^

File: types/index.d.ts [hunk 1]
```diff
@@ -107,7 +107,7 @@ export function combine(...hooks: Hook[]): Hook;
  * Display the current hook context for debugging.
  * {@link https://feathers-plus.github.io/v1/feathers-hooks-common/index.html#Debug}
  */
-export function debug(): Hook;
+export function debug(msg: string, fieldNames?: string[]): Hook;
```
Comment (j2L4e) [inline types/index.d.ts]: fieldNames is a rest param, please use `...fieldNames`. Could you please fix the test, too?

Thanks! :)

Comment (brane53) [reply types/index.d.ts]: Done!

**Comment by j2L4e** [comment]
Thanks! @eddyystop lgtm

**Comment by eddyystop** [comment]
Published as v4.20.1

**Pull request merged**